Makefile

```diff
@@ -36,5 +36,20 @@ $(abspath $(lastword $(MAKEFILE_LIST)))_RECURSE_SUBMAKEFILES := true
 # standard rules generation makefile
 include $(rules_mk_path)
 
+#LOGGING_LIB_DIR=$(wallaroo_path)/utils/logging-lib
```

Comment:
Is this commented out code still needed?

Reply:
Nope, removed now, thanks